metadata.go

```diff
@@ -24,8 +24,9 @@ import (
 
 // StackMetadata has all metadata loaded per stack
 type StackMetadata struct {
-	Name string
-	Path string
+	Name        string
+	Path        string
+	Description string
```

Comment:
`Desc` hahaha ?

Reply:
Sorry that I love typing shit :laughing: